nexus/types/src/deployment/planning_input.rs

```diff
@@ -1306,9 +1344,25 @@ impl PlanningInputBuilder {
             network_resources: OmicronZoneNetworkResources::new(),
             ignore_impossible_mgs_updates_since: Utc::now()
                 - MGS_UPDATE_SETTLE_TIMEOUT,
+            active_nexus_zones: BTreeSet::new(),
+            not_yet_nexus_zones: BTreeSet::new(),
         }
     }
 
+    pub fn set_active_nexuses(
+        &mut self,
+        active_nexus_zones: BTreeSet<OmicronZoneUuid>,
+    ) {
+        self.active_nexus_zones = active_nexus_zones;
+    }
+
+    pub fn set_not_yet_nexuses(
+        &mut self,
+        not_yet_nexus_zones: BTreeSet<OmicronZoneUuid>,
+    ) {
+        self.not_yet_nexus_zones = not_yet_nexus_zones;
+    }
+
```

Comment:
I think these are dups of the functions added at L1458-L1471.

Reply:
Oop, good catch, removed in 1af0b04b